Gemfile

```diff
@@ -63,6 +63,7 @@ gem 'combine_pdf'
 gem 'config'
 gem 'connect_vbms', git: 'https://github.com/adhocteam/connect_vbms', tag: 'v2.1.1', require: 'vbms'
 gem 'csv'
+gem 'datadog'
 gem 'date_validator'
 gem 'ddtrace'
```

Comment:
Is the intent to remove `ddtrace` in this PR as well?

Reply:
Yes. You're quick on the draw. I just removed it 3 minutes ago. Haha.

Reply:
Yeah I saw your commit right after lol